app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    new_comment = Comment.new(author: current_user, post:, text: comment_params)
```

Comment:
- [ ] The syntax error mentioned by the previous reviewer still exits. I suggest you rename the variable `post = Post.find(params[:post_id])` to something else maybe `user_post = Post.find(params[:post_id])` and use this variable on line 4. So it would be ` new_comment = Comment.new(author: current_user, post: user_post, text: comment_params)`. This will clear the syntax error and also make your linters pass 👍. Also remember to change line 11 from `post.id` to `user_post.id` so that you don't encounter any error.

<img width="898" alt="chrome_fMD0CLQdEZ" src="https://user-images.githubusercontent.com/81468675/155516816-022ecb25-ea3d-4fb9-beb6-208804a068f1.png">